### PR TITLE
WIP: Revert to Python 3.7

### PR DESCRIPTION
**Pull request by effigies**
Maybe the problem with #2409 is just nipype + Python 3.8+.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -125,20 +204,18 @@ ENV PATH="/usr/local/miniconda/bin:$PATH" \
...
-                     scipy=1.1.0 \
-                     scikit-learn=0.19.1 \
-                     matplotlib=2.2.2 \
-                     pandas=0.23.4 \
-                     libxml2=2.9.8 \
-                     libxslt=1.1.32 \
-                     graphviz=2.40.1 \
-                     traits=4.6.0 \
-                     zlib; sync && \
+RUN conda install -y python=3.8 \
```
Comment (effigies) [inline Dockerfile:207]:
```suggestion
RUN conda install -y python=3.7 \
```

**Comment by mgxd** [comment]
I wonder if this is the same segfault
```
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 516, in run
    result = self._run_interface(execute=True)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 635, in _run_interface
    return self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 741, in _run_command
    result = self._interface.run(cwd=outdir)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 434, in run
    runtime = self._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/fsl/preprocess.py", line 164, in _run_interface
    self.raise_exception(runtime)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 760, in raise_exception
    ).format(**runtime.dictcopy())
RuntimeError: Command:
bet /scratch/fmriprep_wf/single_subject_100185_wf/func_preproc_task_machinegame_run_02_wf/initial_boldref_wf/enhance_and_skullstrip_bold_wf/n4_correct/ref_sbref_corrected.nii.gz /scratch/fmriprep_wf/single_subject_100185_wf/func_preproc_task_machinegame_run_02_wf/initial_boldref_wf/enhance_and_skullstrip_bold_wf/skullstrip_first_pass/ref_sbref_corrected_brain.nii.gz -f 0.20 -m
Standard output:

Standard error:
Segmentation fault (core dumped)
Return code: 0
```

**Comment by mgxd** [comment]
i'd like to test locally first, but if it ends up being complicated i'm fine with downgrading.

**Comment by effigies** [comment]
Well, these are just experiments, so I'll see if that's the root of it. Won't merge anything.

**Comment by effigies** [comment]
Another option: Can we try FSL 6, if we're using neurodocker builds?

**Comment by mgxd** [comment]
this can be closed since #2409 is in

**Pull request closed without merging**